src/lib/express.js

```diff
@@ -222,7 +222,10 @@ function initSwaggerAPI(app) {
 			},
 			basePath: baseApiPath
 		},
-		apis: config.files.routes.map((route) => path.posix.resolve(route))
+		apis: [
+			'./src/app/**/*.routes.js',
+			'./src/app/**/*.model.js'
+		]
```

Comment:
This removes the resolution from relative to absolute paths.  Not sure how necessary it is, but may effect production deployments.  Also the paths to for routes and models are currently configured in `config/assets.js`.  This duplicates that config to multiple places.  Updating the config to the following should address both of these issues.
```
apis: [
    ...config.files.routes.map((route) => path.posix.resolve(route)),
    ...config.files.model.map((model) => path.posix.resolve(model))
]
```

Reply:
Good plan!